projects/angular/src/popover/dropdown/STYLES.md

```diff
@@ -0,0 +1,34 @@
+# Dropdown
+
+## CSS Custom Properties
+
+| CSS Custom Property                 | Description                                                      |
+| ----------------------------------- | ---------------------------------------------------------------- |
+| --clr-dropdown-active-text-color    | The text color of the active dropdown item.                      |
+| --clr-dropdown-bg-color             | The background color of the dropdown.                            |
+| --clr-dropdown-border-color         | The border color of the dropdown.                                |
+| --clr-dropdown-divider-color        | The color of the divider between dropdown items.                 |
+| --clr-dropdown-divider-border-width | Not used.                                                        |
+| --clr-dropdown-child-border-color   | The border color of child dropdown items.                        |
+| --clr-dropdown-bg-hover-color       | The background color of the dropdown when hovering over an item. |
+| --clr-dropdown-selection-color      | The color of the selected dropdown item.                         |
+| --clr-dropdown-box-shadow           | The box shadow of the dropdown.                                  |
+| --clr-dropdown-text-color           | The text color of the dropdown.                                  |
+| --clr-dropdown-header-color         | The color of the dropdown header.                                |
+| --clr-dropdown-header-font-weight   | Not used.                                                        |
+| --clr-dropdown-item-color           | The color of the dropdown item.                                  |
+| --clr-dropdown-item-font-weight     | Not used.                                                        |
+
+## Classes
+
+| Class                | Description                                       |
+| -------------------- | ------------------------------------------------- |
+| dropdown             | The overall dropdown component.                   |
+| dropdown-toggle      | The toggle element inside the dropdown component. |
+| dropdown-menu        | The dropdown menu container.                      |
+| dropdown-header      | The header element inside the dropdown menu.      |
+| dropdown-item        | The individual items in the dropdown menu.        |
+| dropdown-item-active | The active item in the dropdown menu.             |
```

Comment:
Any idea where this comes from? Can't find it in the code

Reply:
fixed